.github/workflows/lighthouse-report.yml

```diff
@@ -55,7 +55,7 @@ jobs:
 
       - name: Add Lighthouse stats as comment
         id: comment_to_pr
-        uses: marocchino/sticky-pull-request-comment@v2.2.0
+        uses: marocchino/sticky-pull-request-comment@39c5b5dc7717447d0cba270cd115037d32d28443 # v2.2.0
```

Comment:
I can add it part of the PR. Let me know that would work. 

> ok, we can give it a try to pin sha
> 
> please explain where you found the actions/checkout hash, because it's not the one I see associated with the v3 release
> 
> And it doesn't seem to actually exist: [actions/checkout@b0e28b5](https://github.com/actions/checkout/commit/b0e28b5ac45a892f91e7d036f8200cf5ed489415)

Yes, That was a bug I fixed it. 

I have answered https://github.com/facebook/docusaurus/pull/6984#issuecomment-1079832148